docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2037,6 +2061,7 @@ Go to [rules priorities](#rule-priorities) for more details.
 
 - Rules with `$redirect` modifier are not supported by AdGuard Content Blocker, AdGuard for iOS and AdGuard for Safari.
 - `$redirect` in uBlock Origin supports specifying priority, e.g. `$redirect=noopjs:42`. AdGuard does not support it and instead just discards the priority postfix.
+- In AdGuard Browser Extension MV3 extension, allowlist rules with `$redirect` are not supported.
```

Comment:
@105th all of them or is there a case where they work?

Reply:
currently all of them :(